crates/uv/tests/edit.rs

```diff
@@ -5276,6 +5280,9 @@ fn add_index_url() -> Result<()> {
 
         [[tool.uv.index]]
         url = "https://pypi.org/simple"
+
+        [tool.uv.sources]
+        jinja2 = { index = "pytorch" }
```

Comment:
This is a little complicated in this test case because we're in the example "Adding a subsequent index with the same name should replace it."

Is it worth having a simpler test case that's not replacing an existing index at the same time?